### PR TITLE
fix(infiniteGrid): fix image check module in IE9

### PR DESCRIPTION
**Pull request by sculove**
## Issue
#309 

## Details
IE9 couldn't unaware `load` event of images 

## Preferred reviewers
@kishu

### CODE REVIEW

**Comment by mixed** [comment]
LGTM

File: src/infiniteGrid.js [hunk 1]
```diff
@@ -824,8 +824,10 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document, "Outlayer"], function
 				$(e.target).off("load error");
 				checkCount <= 0 && callback && self.core && callback.apply(self);
 			};
+			var $el;
```
Comment (netil) [inline src/infiniteGrid.js]: Is necessary to declare outside of function?

Comment (sculove) [inline src/infiniteGrid.js]: It's not necessary. I want to declare a variable before running `each` method.

Comment (netil) [inline src/infiniteGrid.js]: I think is creating unused closure scope.

Comment (jongmoon) [inline src/infiniteGrid.js]: @netil I think he(@sculove) worries about performance. But It does not make matter anymore. 
http://jindo.dev.naver.com/jsMatch/index.html?d=256&openResult=1

So I agree to remove `var $el`.

But I don't think there will be closure issue. I think $el will be remove from memory after this function is ended.

File: src/infiniteGrid.js [hunk 2]
```diff
@@ -824,8 +824,10 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document, "Outlayer"], function
...
 			$.each(needCheck, function(k, v) {
-				$(v).on("load error", onCheck);
+				$el = $(v);
+				$el.attr("src", $el.attr("src")).on("load error", onCheck);
```
Comment (kishu) [inline src/infiniteGrid.js]: I don't understand this code. Why src attribute assigned it's src???

Comment (sculove) [inline src/infiniteGrid.js]: check this issue https://github.com/naver/egjs/issues/309
and this article. http://garage.socialisten.at/2013/06/how-to-fix-the-ie9-image-onload-bug/

**Comment by kishu** [comment]
LGTM

File: src/infiniteGrid.js (hunk 2 above)
Comment (jongmoon) [inline src/infiniteGrid.js]: @kishu, @sculove I didn't notice it! New Information to me ~!! Thank you!

**Pull request merged**